pom.xml

```diff
@@ -60,6 +106,17 @@
             <artifactId>javax.annotation-api</artifactId>
             <version>${javax.annotation.version}</version>
         </dependency>
+        <!-- SPRING -->
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
idziemy w spring czy spring boot?

Reply:
zależności kopiowałem na wzór tego co było wcześniej, nie analizowałem jak to inaczej ograć, jak masz inną koncepcje to można pewnie drugim PR to pozmieniać

Reply:
nie mam, po prostu się zastanawiam czy nie lepiej korzystać ze spring boota, ale to w przyszłości możemy zmienić jak faktycznie zaczniemy z tego korzystać